Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -258,13 +258,13 @@ def fetch_indicators_from_file(args):
     return human_readable, domain_context, indicator_list
 
 
-def main():
+def main():  # pragma: no cover
```

Comment:
Why did you add here the no cover?

Reply:
added more tests, not sure why pragma: no cover anymore